raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -63,50 +71,71 @@ contract NettingChannelContract {
         return data.addressAndBalance();
     }
 
+    /// @notice Get the amount one partner has transferred to the other partner.
+    /// @param participant The address of one partner.
+    /// @return The amount one partner has transferred to the other.
     function transferredAmount(address participant) constant returns (uint) {
         return data.transferredAmount(participant);
     }
 
+    /// @notice Close the channel. Can only be called by a participant in the channel.
+    /// @param theirs_encoded The last transfer recieved from our partner.
+    /// @param ours_encoded The last transfer sent to our partner.
     function close(bytes theirs_encoded, bytes ours_encoded) {
         data.close(msg.sender, theirs_encoded, ours_encoded);
         ChannelClosed(msg.sender, data.closed);
     }
 
+    /// @notice Dispute the state after closing, called by the counterparty (the
+    ///         participant who did not close the channel).
+    /// @param theirs_encoded The transfer the counterparty believes is the valid
+    ///                       state of the first participant.
     function updateTransfer(bytes theirs_encoded) {
         data.updateTransfer(msg.sender, theirs_encoded);
         TransferUpdated(msg.sender, block.number);
     }
 
+    /// @notice Unlock a locked transfer.
```

Comment:
Guess it should have had but I missed it during the review. It's all right though, can be added in a subsequent commit.